src/modules/awake/Awake/Properties/Resources.resx

```diff
@@ -121,7 +121,7 @@
     <value>Checked</value>
   </data>
   <data name="AWAKE_EXIT" xml:space="preserve">
-    <value>Exit</value>
+    <value>Close</value>
```

Comment:
This is exit, not close.

closing implies it still runs

Reply:
Nope. Microsoft is very clear on that. You must be mistaken with closing a window.

Reply:
That is for writing documents, not everything applies to an application.

Reply:
please revert this one

Reply:
Done